source/deployment-options/docker/upgrading-wazuh-docker.rst

```diff
@@ -227,6 +227,19 @@ To upgrade your deployment keeping your custom docker-compose files, do the foll
                   - DASHBOARD_USERNAME=kibanaserver
                   - DASHBOARD_PASSWORD=kibanaserver
 
+#. Replace these files with the ones from the v|WAZUH_CURRENT_DOCKER| version of wazuh-docker repository in your own deployment.
```

Comment:
Update the following:
`version of wazuh-docker`
With:
`version of the wazuh-docker`

Reply:
done